rapier/src/lib.rs

```diff
@@ -70,6 +71,7 @@ pub struct ColliderHandle(geometry::ColliderHandle);
 impl Plugin for RapierPlugin {
     fn build(&self, app: &mut App) {
         app.add_plugin(heron_core::CorePlugin)
+            .add_plugin(TimePlugin)
```

Comment:
If application adds `TimePlugin`, this addition breaks time delta. You can reproduce with following application:
```rust
use bevy::app::ScheduleRunnerSettings;
use bevy::log::LogPlugin;
use bevy::prelude::*;
use std::time::Duration;
use bevy::time::TimePlugin;

fn main() {
    App::new()
        .insert_resource(ScheduleRunnerSettings::run_loop(Duration::from_secs(1)))
        .add_plugins(MinimalPlugins)
        .add_plugin(LogPlugin)
        // .add_plugin(TimePlugin) // uncomment and see the difference
        .add_system(log_time)
        .run();
}

fn log_time(time: Res<Time>) {
    info!("time: {:?}", time.delta());
}
```

Outputs:

Without extra `TimePlugin`:
```
2022-11-10T00:35:32.996089Z  INFO bevy_test: time: 0ns
2022-11-10T00:35:34.007733Z  INFO bevy_test: time: 1.0119971s
2022-11-10T00:35:35.012710Z  INFO bevy_test: time: 1.0049682s
2022-11-10T00:35:36.018760Z  INFO bevy_test: time: 1.0060325s
```

With extra `TimePlugin`:
```
2022-11-10T00:33:18.515528Z  INFO bevy_test: time: 1µs
2022-11-10T00:33:19.527569Z  INFO bevy_test: time: 1.1µs
2022-11-10T00:33:20.529827Z  INFO bevy_test: time: 1.7µs
2022-11-10T00:33:21.540661Z  INFO bevy_test: time: 1.1µs
```

I'd suggest to remove this line. Applications should add all required plugins before adding heron plugins.